src/ctype/CTypeMetadata.spec.ts

```diff
@@ -0,0 +1,46 @@
+import CType from './CType'
+import ICType from '../types/CType'
+import CTypeMetadata from './CTypeMetadata'
+import * as CTypeUtils from './CTypeUtils'
+import { CTypeWrapperMetadata } from './CTypeSchema'
+import ICTypeMetadata from '../types/CTypeMetadata'
+
+jest.mock('../blockchainApiConnection/BlockchainApiConnection')
```

Comment:
Why do you need the mock? I don't see a call to ctype.store or similar

Reply:
Done!